client/src/containers/Topic/Topic/TopicData/TopicData.jsx

```diff
@@ -910,7 +928,13 @@ class TopicData extends Root {
                     colName: 'Date',
                     type: 'text',
                     cell: (obj, col) => {
-                      return (<TimeAgo date={Date.parse(obj[col.accessor])} title={obj[col.accessor]}/>);
+                      const isoDate = obj[col.accessor]
+                      const TimeAgoComp = <TimeAgo date={Date.parse(isoDate)} title={""}/>
+                      return (
+                          <Tooltip arrow title={!this.state.showRelativeTimes ? TimeAgoComp : isoDate} interactive>
```

Comment:
the `interactive` prop enables the selection of the tooltip's value. This ensures that the tooltip's content can be copied.